cmd/controller/app/controller.go

```diff
@@ -23,7 +23,7 @@ import (
 	"sync"
 	"time"
 
```

Comment:
All the changes in this commit are mechanical. Produced by the script in the previous commit.